### ISSUE TITLE
Error when importing remote data into influxdb

### ISSUE THREAD

**Issue by samfaunt** [lux4rd0/weatherflow-collector#22]
When trying to import remote data into influxdb using the collector getting the following error.. 

 {"error":"unable to parse 'weatherflow_rapid_wind,collector_key=xxxxx,collector_type=remote-socket,device_id=xxxxx,elevation=87.xxxxx,source=collector,hub_sn=HB-xxxxx,latitude=-34.xxxxx,longitude=138.xxxxx,public_name=,serial_number=ST-xxxxxx,station_id=,station_name=,timezone= time_epoch=1702215898000,wind_speed=0.51,wind_direction=135 1702215898000000000': missing tag value"}

**Comment by lux4rd0** [comment]
Hi @samfaunt  - you're missing these details:

public_name=,serial_number=ST-xxxxxx,station_id=,station_name=,timezone=

Did you build out the docker-compose script using the "generate_docker-compose.sh" 

That should have pulled in all of the details needed for the script. Since you have things like elevation and hub_sn in there, it seems like you did - I'm just not pulling in the right details. Do you have more than one device on your account? Happy to troubleshoot or make this easier to use!

**Comment by samfaunt** [comment]
Yep, used your docker-compose script to build the import scripts.  I have a second site in my account with a device that is offline, but i also specified the station with the environment variables that are available, but those errors still persist.  Let me know how you want to proceed with troubleshooting. 

Thanks!

**Comment by samfaunt** [comment]
bump @lux4rd0

**Comment by lux4rd0** [comment]
@samfaunt, could you briefly share an API key with me? I'm rebuilding the entire application, and if I can see how I'm pulling the data with your specific account, I should be able to make a fix pretty quickly. Don't share it online - but perhaps email?? d a v e at p u l p f r e e dot o r g 

Thanks!

**Comment by samfaunt** [comment]
Yep @lux4rd0 will email you shortly. 

Cheers Sam

**Comment by jake9190** [comment]
Not sure if this was ever resolved, but I ran into the same issue (without any changes) last night and my remote-rest calls stopped working. I'm still running 3.5.3 since I have InfluxDB 1.11, but I think the issue is related to the json query:

`eval "$(echo "${line}" | jq -r '. | to_entries | .[0,2,3,4,5,6,7,8,11] | .key + "=" + "\"" + ( .value|tostring ) + "\""')"`

In my case, `elevation` is 1, and `station_name` is 9 so they are not being populated, and therefore the influxdb write call fails due to the tags missing.

For now I changed this to:

`eval "$(echo "${line}" | jq -r '. | to_entries | .[] | .key + "=" + "\"" + ( .value|tostring ) + "\""')"`

manually within the container, and will probably fork to persist this until I can migrate to InfluxDB 2
